controlplane/src/core/util.ts

```diff
@@ -160,3 +160,13 @@ export const hasLabelsChanged = (prev: Label[], cur: Label[]): boolean => {
       .join(',')
   );
 };
+
+// checks if the user has the right roles to perform the operation.
+export const checkUserAccess = ({ rolesToBe, userRoles }: { rolesToBe: string[]; userRoles: string[] }) => {
```

Comment:
Can we use drizzle types to make roles everywhere type-safe?

Reply:
Defined a type, as accesisng the enum value is like memberRoleEnum.enumValues[0]